SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DeviceInformationHelper.cs

```diff
@@ -95,7 +95,7 @@ protected override string GetOsVersion()
 
         protected override string GetAppVersion()
         {
-#if NET45
+#if NET461
```

Comment:
Does it work? We were not using the third digit before so just making sure.

Reply:
Yes